package.json

```diff
@@ -13,7 +13,9 @@
     "type-check:watch": "yarn run type-check -- --watch",
```

Comment:
Can we please bump the version to `"version": "2.1.0"` in this PR and I can release.

Reply:
Done